### PR TITLE
CEDS-1919 update final screen

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/models/responses/FlashKeys.scala [hunk 1]
```diff
@@ -18,5 +18,6 @@ package models.responses
 
 object FlashKeys {
   val lrn: String = "LRN"
+  val decType: String = "DECTYPE"
```
Comment (PatrykRudnicki) [inline app/models/responses/FlashKeys.scala]: I'll use as value `DecType` the same approach like `expiryDate` not `lrn`

Comment (SteveSugden) [reply app/models/responses/FlashKeys.scala]: Good point.  I should be able to change them all to camelCase.   Will (hopefully) flush out anywhere that is not using these constants (like confirmation page wasn't)

Comment (SteveSugden) [reply app/models/responses/FlashKeys.scala]: 2nd thought - initial case.  And "LRN" is probably OK as it's an abbreviation.

Comment (PatrykRudnicki) [reply app/models/responses/FlashKeys.scala]: Yeah, `lrn` is correct

File: test/views/declaration/SubmissionConfirmationPageViewSpec.scala [hunk 2]
```diff
@@ -30,65 +31,51 @@ import views.tags.ViewTest
...
-      messages must haveTranslationFor("supplementary.confirmation.explanation.linkText")
-      messages must haveTranslationFor("supplementary.confirmation.submitAnotherDeclaration")
-      messages must haveTranslationFor("supplementary.confirmation.rejection.header")
-    }
-
-    "display header" in {
-      view.select("article>div.govuk-box-highlight>h1").text() mustBe "supplementary.confirmation.header"
-      view.select("article>div.govuk-box-highlight>p").text() mustBe "-"
+    "display header with default" in {
+      val highlightBox = view.selectFirst("article>div.govuk-box-highlight")
```
Comment (bambuchaAdm) [inline test/views/declaration/SubmissionConfirmationPageViewSpec.scala]: quite agresive matcher

**Pull request merged**